src/components/Form/formFunctions.ts

```diff
@@ -120,6 +120,48 @@ export const onChangeRadioGroup = (
     };
 };
 
+export const valueOnChangeRadioGroup = (
```

Comment:
Whats the difference between this function and the one above (onChangeRadioGroup). Is it possible to combine these two functions?

Reply:
So one sets the radio group to true or false, the other sets it to the value of the label. This potentially could be combined but I don't think it is worth it, as the first one has a very specific use in Add Clients Page

Reply:
what do you think about changing radio group so that the `value` doesn't have to be a `string` and hence we can just always set it to the value of the label? (i.e. if `value` can also be `boolean`). This won't be in the same ticket btw, just asking for your opinion.

Reply:
I'll have a look, don't have a problem with the idea for sure. However thinking more I could potentially adapt the add clients code to fit my function? Not sure though but will have a mess around for sure.

Reply:
sounds good :)